lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -286,24 +286,28 @@ export async function getUpdatedPackageFiles(
   const updatedArtifacts: FileChange[] = [];
   const artifactErrors: ArtifactError[] = [];
   for (const packageFile of updatedPackageFiles) {
-    const manager = packageFileManagers[packageFile.path];
     const updatedDeps = packageFileUpdatedDeps[packageFile.path];
-    const updateArtifacts = get(manager, 'updateArtifacts');
-    if (updateArtifacts) {
-      const results = await updateArtifacts({
-        packageFileName: packageFile.path,
-        updatedDeps,
-        // TODO #22198
-        newPackageFileContent: packageFile.contents!.toString(),
-        config,
-      });
-      if (is.nonEmptyArray(results)) {
-        for (const res of results) {
-          const { file, artifactError } = res;
-          if (file) {
-            updatedArtifacts.push(file);
-          } else if (artifactError) {
-            artifactErrors.push(artifactError);
+    const managers = packageFileManagers[packageFile.path];
+    if (managers !== undefined) {
```

Comment:
We prefer using `is.undefined` in such cases.
@viceice we use `!== undefined` in some places whereas `is.undefined` at other..
We should be consisitent right?

Reply:
Good point, thanks! I went for `is.nonEmptyArray` to avoid the negation